components/amount-buttons.jsx

```diff
@@ -97,14 +98,40 @@ var AmountButtons = React.createClass({
   },
   validate: function() {
     var valid = false;
+    var errorMessage = "";
     if (this.props.amount) {
-      valid = true;
+      if (parseInt(this.props.amount, 10) < parseInt(this.props.currency.minAmount, 10)) {
+        errorMessage = this.getIntlMessage('donation_min_error');
+      } else {
+        valid = true;
+      }
+    } else {
+      errorMessage = this.getIntlMessage('please_select_an_amount');
     }
     this.setState({
-      valid: valid
+      valid: valid,
+      errorMessage: errorMessage
     });
     return valid;
   },
+  renderErrorMessage: function() {
+    if (this.state.errorMessage === this.getIntlMessage('donation_min_error')) {
+      return (
+        <FormattedMessage
+          message={this.state.errorMessage}
+          minAmount={
+            <FormattedNumber
+              maximumFractionDigits={2}
+              value={this.props.currency.minAmount}
+              style="currency"
+              currency={this.props.currency.code || "usd"}
+            />
+          }
+        />
+      );
+    }
+    return this.state.errorMessage;
```

Comment:
Render the FormattedMessage is needed, or just return the error. The error could also be "" which means don't display anything or the default error.